presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.234
+=============
+
+.. warning::
+
+   There is a regression in the scheduler that has not been fixed in this release. It has been temporarily mitigated by setting
+   the default value of the configuration property ``use-legacy-scheduler`` to ``true``. Please keep the legacy scheduler enabled
+   until the issue is fully resolved.
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix an issue where ``IGNORE NULLS`` is not respected for window functions.
+* Improve coordinator reliability by adding support to cap the number of total tasks running in a coordinator by pausing scheduling.
+  This can be enabled by the configuration property ``experimental.max-total-running-task-count-to-not-execute-new-query``.
+* Improve the scale writer heuristics by considering overall producer buffer utilization. This can be enabled by the session property
+  ``optimized_scale_writer_producer_buffer`` and the configuration property ``optimized-scale-writer-producer-buffer``.
+* Improve end point ``v1/resourceGroupState`` to supporting filtering of resource groups that are dynamically generated.
+* Improve connection pooling to avoid running out of sockets.
+* Add :ref:`khyperloglog_type` type and related functions.
+* Add support for forcing streaming exchange for Mark Distinct. This can be enabled by the session property ``use_stream_exchange_for_mark_distinct``
```

Comment:
"Add support for forcing streaming exchange for Mark Distinct" -> "Add support for forcing streaming exchange for Mark Distinct even if materialized exchange is enabled."